### PR TITLE
Refactoring for better consistency with other adapters

### PR DESCRIPTION
**Pull request by BenjaminRueth**
this pull request addresses the comment https://github.com/precice/tutorials/pull/10#issuecomment-428876263.

### CODE REVIEW

File: tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py [hunk 1]
```diff
@@ -128,11 +123,9 @@ def fluxes_from_temperature_full_domain(F, V, k):
 coupling_boundary = TopBoundary()
 bottom_boundary = BottomBoundary()
 
-coupling = Coupling(config_file_name, solver_name)
-coupling.set_coupling_mesh(mesh, coupling_boundary, coupling_mesh_name)
-coupling.set_read_field(read_data_name, u_D_function)
-coupling.set_write_field(write_data_name, f_N_function)
-coupling.initialize_data()
+precice = Adapter()
```
Comment (uekerman) [inline tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py]: This is weird. You call it `Adapter` and then `precice`. I would at least change `precice` here to `preciceAdapter`. Do you agree?

Comment (uekerman) [inline tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py]: This is weird. You call it `Adapter` and then `precice`. I would at least change `precice` here to `preciceAdapter`. Do you agree?

File: tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py [hunk 2]
```diff
@@ -153,33 +146,31 @@ def fluxes_from_temperature_full_domain(F, V, k):
...
 file_out = File("Solid/VTK/%s.pvd" % solver_name)
 
-while coupling.is_coupling_ongoing():
+while precice.is_coupling_ongoing():
 
     # Compute solution
     solve(a == L, u_np1, bcs)
 
     # Dirichlet problem obtains flux from solution and sends flux on boundary to Neumann problem
     fluxes = fluxes_from_temperature_full_domain(F_known_u, V, k)
```
Comment (uekerman) [inline tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py]: Could this line also go into the adapter?

File: tutorials/HT/partitioned-heat/fenics-fenics/heat.py [hunk 3]
```diff
@@ -173,10 +171,10 @@ def fluxes_from_temperature_full_domain(F, V):
 
 if problem is ProblemType.DIRICHLET:
```
Comment (uekerman) [inline tutorials/HT/partitioned-heat/fenics-fenics/heat.py]: could this go into `Adapter.initialize`?

Comment (BenjaminRueth) [reply tutorials/HT/partitioned-heat/fenics-fenics/heat.py]: This is probably something we should discuss. There are good reasons for both possible ways.

File: tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py (hunk 2 above)
Comment (BenjaminRueth) [reply tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py]: see https://github.com/precice/fenics-adapter/issues/3, if you mean line 167

File: tutorials/HT/partitioned-heat/fenics-fenics/heat.py (hunk 3 above)
Comment (BenjaminRueth) [reply tutorials/HT/partitioned-heat/fenics-fenics/heat.py]: Maybe this discussion also belongs here https://github.com/precice/fenics-adapter/issues/3

File: tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py (hunk 1 above)
Comment (BenjaminRueth) [reply tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py]: I agree :) I will rename everything: ```precice_adapter = PreciceAdapter()```

Comment (BenjaminRueth) [reply tutorials/CHT/flow-over-plate/buoyantPimpleFoam-fenics/Solid/heat.py]: same https://github.com/precice/fenics-adapter/pull/6#discussion_r226974367

**Comment by BenjaminRueth** [comment]
We also have to synchronize the tutorials. See https://github.com/precice/tutorials/pull/12

**Pull request merged**